inclusive-design/idrc
# feat: add ideas section

## Description

This PR adds the ideas section for long form writing.

- [ ] Ideas view
  - [ ] Ideas grid with featured and fallback images
  - [ ] Featured items
- [ ] Single idea view
  - [ ] Featured images
  - [ ] Inline image alignments (float left, float right, center)

## Steps to test

1. Navigate to `/ideas/` on the deploy preview.
2. View the example idea.

**Expected behavior:** Implementation matches Figma designs (see #41).

## Additional information

Not applicable.

## Related issues

Resolves #41.
